instrumentation/instaawssdk/invoke.go

```diff
@@ -0,0 +1,66 @@
+package instaawssdk
+
+import (
+	"bytes"
+	"encoding/base64"
+	"encoding/json"
+
+	otlog "github.com/opentracing/opentracing-go/log"
+
+	"github.com/aws/aws-sdk-go/aws/request"
+	"github.com/aws/aws-sdk-go/service/lambda"
+	instana "github.com/instana/go-sensor"
+	"github.com/opentracing/opentracing-go"
+	"github.com/opentracing/opentracing-go/ext"
+)
+
+func StartInvokeSpan(req *request.Request, sensor *instana.Sensor) {
```

Comment:
I'd suggest we avoid using generic "invoke" in our API. There are too many things that can be invoked :) How about `StartInvokeLambdaSpan`?

Reply:
Agree, will change it.